log/log.go

```diff
@@ -25,26 +24,11 @@ import (
 	"strconv"
 	"strings"
 
+	"gopkg.in/alecthomas/kingpin.v2"
+
```

Comment:
Please remove the extra line and list all external package dependencies in one block.